### DISCUSSION TITLE
Is leptos standalone?

### DISCUSSION THREAD

**Discussion by Gadersd** [leptos-rs/leptos#1226] [Q&A] [answered]
Hello, I am new to web development in general and am excited for leptos as it seems so much safer and reliable than using javascript. What features for building web apps are missing from leptos? Would I need actix/axum for some scenarios, or is it possible to do everything in leptos? I looked through the docs for leptos and couldn't find anything regarding cookies. Would I need to hook leptos into actix or axum to handle cookies?

**Comment by gbj** [comment, marked as answer]
Leptos is an application framework, not a server framework. It sits on top of a server framework like Actix or Axum.

If you're coming from JS world and this is a useful analogy: Leptos is React or Vue or Svelte or Solid, not Express or Fastify.

See more discussion in the docs [here](https://leptos-rs.github.io/leptos/server/26_extractors.html#server-frameworks).
